src/api/spec/factories/packages.rb

```diff
@@ -28,5 +28,14 @@
         Suse::Backend.put("/source/#{URI.escape(package.project.name)}/#{URI.escape(package.name)}/_service", '<service/>')
       end
     end
+
+    factory :package_with_failed_comment_attribute do
+      after(:create) do |package|
+        attribute_type = AttribType.find_by_name("OBS:ProjectStatusPackageFailComment")
+        attrib = build(:attrib, attrib_type: attribute_type, package: package)
```

Comment:
Because the validation depends on the `attrib_type`, see:
https://github.com/eduardoj/open-build-service/blob/c56cc090c25de490f069ed2d12b0a4e4ee09b25a/src/api/app/models/attrib.rb#L135

If you create an `Attrib` with `OBS:ProjectStatusPackageFailComment` type you'll get this error:

```
ActiveRecord::RecordInvalid:
       Validation failed: Values has 0 values, but only 1 are allowed
```